### DISCUSSION TITLE
Chat for those who doesn't have ltmeat flood working or ext hang3r

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1293] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
So basically you just run the amount of tabs for ltmeat flood https://chrose.netlify.app/detail/ltmeat%20flood/ and then just run it and do the instructions. After goguardian has been frozen you want to activate ext hang3r and then it just lasted for like 2 hours for me. LET ME KNOW HOW LONG IT LASTES FOR YOU ( probably because of the amount of tabs)

**Comment by ghost** [comment]
Also I will be revealing proxies to those who need it! Just alert me an discussion and Ill respond as fast as possible.

**Comment by ghost** [comment]
Some may not work so Ill find some more :)

**Comment by py660** [comment]
This belongs in #help or #polls, not #exploits.

**Reply by ghost** [reply]
sry
